README.md

```diff
@@ -72,14 +72,8 @@ import { withComponent } from 'skatejs';
 const Component = withComponent();
 
 class GreetingComponent extends Component {
-  renderer (renderRoot, render) {
-    renderRoot.innerHtml = '';
-    renderRoot.appendChild(render());
-  }
-  render () {
-    const el = document.createElement('span');
-    el.innerHTML = 'Hello, <slot></slot>!';
-    return el;
+  render() {
+    return 'Hello, <slot></slot>!';
```

Comment:
I meant to discuss this with you. Do you think this is a valuable change? It felt very right to do.

Reply:
yup it's definitely good to have some default just by installing skate with any deps :) 👍